### DISCUSSION TITLE
Correlate traces in APM with logs

### DISCUSSION THREAD

**Discussion by zellen** [signalfx/splunk-otel-collector#5506] [Q&A]
We are using the splunk-otel-collector to send logs to Splunk enterprise and traces to Splunk Obsevability Cloud. A connection has been made so that we can see the logs in Log Observer.

What I want to know is how to correlate our logs with our traces. As of now the logs section when looking at a trace is empty: 
![image](https://github.com/user-attachments/assets/51bf0c55-5000-4515-9e93-d7468fd4db0c)

Is this possible to achieve only using the collector or do we have to do anything else?

**Comment by pjanotti** [comment]
What is generating the logs? This typically is done by the application instrumentation itself that injects the TraceID into the log statements, the collector just forwards these logs to the backend.
